app/js/services/NotificationService.js

```diff
@@ -1,26 +1,24 @@
-function NotificationService($window, $log, growl) {
-  'ngInject';
-  $log = $log.getInstance('NotificationService', true);
-  $log.log('');
+function NotificationService($window, $log, $rootScope) {
+  "ngInject";
+  $log = $log.getInstance("NotificationService", true);
+  $log.log("");
 
-  const service = {
-    enabled: true, 
-    disable:()=>{  service.enabled = false; },
-  	notify:function(args){
-      
-      let settings = angular.extend({}, {show:true, config:{}},args);
-  		if(service.enabled && settings.show) growl[settings.type](settings.message, settings.config);
+  const service = {};
 
-    }
+  /**
+   * Component calls this function to emit a global event that trigger a global notification
+   * @param  {object} args - An object of notification properties
+   *              | - {string} type - type of notification (i.e. "success")
+   *              | - {string} message - message of notification, can contain HTML
+   */
+  service.notify = args => {
+    $rootScope.$emit("TRIGGERED_NOTIFICATION", args);
```

Comment:
Here I would usually say something about avoiding mucking about with `$rootScope` and `$emit`, but since you are replacing a global notification library this might actually be the most efficient approach.